### ISSUE TITLE
Peer review count is out of order for screen readers when creating a course

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#2501]
![image](https://user-images.githubusercontent.com/1316902/53041579-334d6080-3439-11e9-88fa-04b1852b61ba.png)
Right now, a screenreader will read the above as 1, 3, 2 instead of 1, 2, 3. This is because of how the elements are ordered on the page. If possible, these numbers should be read in order.

**Comment by mariahajmal** [comment]
@bwreid  Hi, I am an outreachy applicant. Can I work on this?

**Comment by MasterIfeanyi** [comment]
I believe this is no longer an issue, 7 years later. code has likely moved and changed. 

Please close this issue @ragesoss
